libraries/iio/iio.c

```diff
@@ -819,12 +819,6 @@ static uint32_t bytes_to_samples(struct iio_interface *intf, uint32_t bytes)
 static ssize_t iio_transfer_dev_to_mem(const char *device, size_t bytes_count)
 {
 	struct iio_interface *iio_interface = iio_get_interface(device);
-
-	if (iio_interface->dev_descriptor->transfer_dev_to_mem)
```

Comment:
Shouldn't library changes be subject for another PR? Since these changes might influence also other projects than ad7134.

Reply:
Only this projects was still using this functions. The scope was to remove them so the project has to be updated. 
Anyway, the changes are in separated commits.